open-irc-links/README.md

```diff
@@ -0,0 +1,13 @@
+# open-irc-links
+
+
+## What it does
+
+This add-on sets default client for opening IRC links using protocol_handlers.
+
+Whenever any link with IRC protocol is clicked it opens with pre-defined URI template.
+For example click [here](irc://irc.freenode.net/drupal).
```

Comment:
Please change to "For example, click [this link to irc://irc.freenode.net/drupal](irc://irc.freenode.net/drupal) and the protocol handler opens the Drupal channel in mibbit.com."